app/controllers/response_controller.rb

```diff
@@ -211,13 +217,21 @@ def create
     redirect_to :controller => 'response', :action => 'saving', :id => @map.map_id, :return => params[:return], :msg => msg, :error_msg => error_msg, :save_options => params[:save_options]
   end
 
+  #E1600
+  #Added paramps[:return] value for 'SelfReviewResponseMap'
   def saving
     @map = ResponseMap.find(params[:id])
+    if(@map.type == "SelfReviewResponseMap")
```

Comment:
I checked if I could make a change to accommodate this return value in view, but I think it would be too messy as the response view is already complicated and making changes there would make reading the code more difficult. I think it is better to have it in controller itself for maintenance purposes